test/test_common/environment.cc

```diff
@@ -264,7 +264,11 @@ const std::string& TestEnvironment::temporaryDirectory() {
 
 std::string TestEnvironment::runfilesDirectory(const std::string& workspace) {
   RELEASE_ASSERT(runfiles_ != nullptr, "");
-  return runfiles_->Rlocation(workspace);
+  auto path = runfiles_->Rlocation(workspace);
+#ifdef WIN32
+  path = std::regex_replace(path, std::regex("\\\\"), "/");
```

Comment:
Isn't `\\` on Windows a file share? Will that work as expected replacing with a single `/`?

Reply:
> Isn't \\ on Windows a file share?

The output of this will be something like `C:\\Windows\\test` and i transform it  -> `C:/windows/test`

Reply:
ah, so the `\\` is literally in the string for regular paths.